### ISSUE TITLE
Handle multiple editions

### ISSUE THREAD

**Issue by captn3m0** [captn3m0/what-to-read#2]
Often, the edition returned by goodreads is hardback which is costlier on amazon. In some cases, if both editions share the ISBN, it works out. But very often, the cheaper paperback has a different ISBN, which means that it does not show up in the amazon offers list.

So, we are either stuck with a costly hardback edition, or in some cases "unavailable", when in reality there is a cheaper version available.

Find a way.

**Comment by captn3m0** [comment]
Need to find a way to see "other editions" of the book on Amazon. That would give me a way to see prices of all 3, and I can return data for all three accordingly (probably via split rows).

**Comment by poluruprvn** [comment]
http://www.worldcat.org/affiliate/webservices/xisbn/app.jsp

**Comment by captn3m0** [comment]
That looks interesting. One issue I'm currently facing is that the app makes too many queries and just hangs on heroku.

**Comment by poluruprvn** [comment]
Is that because of the limit set by the amazon API ?

**Comment by captn3m0** [comment]
No, the app works perfectly fine on my local machine. Its just the Heroku instance doesn't have enough resources. I might move to queues or maybe my VPS.
